src/plugin.ts

```diff
@@ -40,6 +42,21 @@ export class PluginTypeEnvironment extends TypeEnvironment {
         return this.getType(this.checker.lookupTypeAt(n, this.sinapSourceFile));
     }
 
+    private getFunctionSignatures(name: string, node: ts.Node) {
+        const functionSymbol = this.checker.getSymbolsInScope(node, ts.SymbolFlags.Function)
+            .filter((a) => a.name === name)[0];
+        if (functionSymbol === undefined) {
+            throw new Error(`function "${name}" not found`);
```

Comment:
What is the advantage of throwing an error versus throwing a string? If there is no benefit, then I think we should go with the simpler solution.